include/baresip.h

```diff
@@ -1381,7 +1381,7 @@ struct stream *audio_strm(const struct audio *au);
 uint64_t audio_jb_current_value(const struct audio *au);
 int  audio_set_bitrate(struct audio *au, uint32_t bitrate);
 bool audio_rxaubuf_started(const struct audio *au);
-int  audio_start(struct audio *a);
+int  audio_update(struct audio *a);
```

Comment:
this will break the API. I think we should try to avoid breaking the API in the future ...

Perhaps keep audio_start() and add audio_update() ?

Reply:
The reason for this PR was to drop all unnecessary redundancy about starting/updating the audio streams.
Also I want to make video and audio API similar.
I'll try to find a way to keep the backward compatibility with minimal redundancy.